src/System.Text.Encoding/tests/UTF7Encoding/UTF7EncodingGetMaxCharCount.cs

```diff
@@ -2,39 +2,28 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class UTF7EncodingGetMaxCharCount
     {
-        private readonly RandomDataGenerator _generator = new RandomDataGenerator();
+        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
At some point we should really go through the tests and remove the usage of this random generator.  Since we're passing in a fixed seed, we're always going to get the same resulting value, and in a case like how it's used here, we'd be better off just picking the values we want to test and using them.